libs/coin-evm/src/api/gasTracker/ledger.ts

```diff
@@ -16,21 +20,14 @@ type GasTracker = {
 // Shouldn't this be a dynamic / remote config? For example if there is an
 // update of the explorer backend to support EIP1559, we should be able to
 // update this config without having to release a new version of the app?
-const currencyIdGasTrackerMap = new Map<CryptoCurrency["id"], GasTracker>([
-  ["avalanche_c_chain", { explorerId: "avax", compatibilty: { eip1559: false } }],
-  ["bsc", { explorerId: "bnb", compatibilty: { eip1559: false } }],
-  ["ethereum", { explorerId: "eth", compatibilty: { eip1559: true } }],
-  ["ethereum_classic", { explorerId: "etc", compatibilty: { eip1559: false } }],
-  ["polygon", { explorerId: "matic", compatibilty: { eip1559: true } }],
-  ["ethereum_ropsten", { explorerId: "eth_ropsten", compatibilty: { eip1559: true } }],
-  ["ethereum_goerli", { explorerId: "eth_goerli", compatibilty: { eip1559: true } }],
-  /**
-   * FIXME: for tests, to be removed when ethereum_as_evm_test_only and
-   * polygon_as_evm_test_only are removed from
-   * libs/ledgerjs/packages/cryptoassets/src/currencies.ts
-   */
-  ["ethereum_as_evm_test_only", { explorerId: "eth", compatibilty: { eip1559: true } }],
-  ["polygon_as_evm_test_only", { explorerId: "matic", compatibilty: { eip1559: true } }],
+const currencyIdGasTrackerMap = new Map<LedgerExplorerId, GasTracker>([
```

Comment:
To ease understanding

```suggestion
const explorerIdGasTrackerMap = new Map<LedgerExplorerId, GasTracker>([
```

Reply:
Nice catch 👀 
Will update 👌

Reply:
Done